pkgs/top-level/all-packages.nix

```diff
@@ -1105,6 +1105,11 @@ in
 
   apksigcopier = callPackage ../development/tools/apksigcopier { };
 
+  apksigner = import ../development/tools/apksigner {
+    inherit runCommand makeWrapper jre;
```

Comment:
```suggestion
```

Reply:
Thanks. but I had to revert that change again. it's not needed for `callPackage`, but it is for `import`.

Reply:
Then you should use `callPackage`.

Reply:
Yes, I should. Thanks!

I'll blame it on lack of sleep :sweat_smile: